lib/administrate/page/nested_collection.rb

```diff
@@ -0,0 +1,11 @@
+require 'administrate/page/collection'
+
+module Administrate
+  module Page
+    class NestedCollection < Page::Collection
+      def attribute_names
+        options.fetch(:collection_attributes, dashboard.collection_attributes)
```

Comment:
How about the following? That way you only call `dashboard.collection_attributes` lazily, when there is no `:collection_attributes` available:

```suggestion
        options.fetch(:collection_attributes) { dashboard.collection_attributes }
```